polkadot/xcm/docs/src/cookbook/relay_token_transactor/mod.rs

```diff
@@ -0,0 +1,35 @@
+//! # Relay asset transactor
+//!
+//! This example shows how to configure a parachain to only deal with the relay chain token.
+//!
+//! The first step is using the [`xcm_builder::CurrencyAdapter`] to create an `AssetTransactor` that
+//! can handle the relay chain token.
+#![doc = docify::embed!("src/cookbook/relay_token_transactor/parachain/xcm_config.rs", asset_transactor)]
+//!
+//! The second step is to configure `IsReserve` to recognize the relay chain as a reserve for its
+//! own asset.
+//! With this, you'll be able to easily get derivatives from the relay chain by using the xcm
+//! pallet's `transfer_assets` extrinsic.
```

Comment:
"Derivative" can have a lot of meanings, IMO this is more clear:
```suggestion
//! With this, you'll be able to easily mint a derivative asset, backed one-to-on from the Relay Chain, by using the xcm
//! pallet's `transfer_assets` extrinsic.
```